README.md

```diff
@@ -17,6 +19,18 @@ npm start
 ```
 The command prompt `KnowSheet> ` will appear.
 
+... or evaluate one expression from the command-line:
+```bash
+echo '(expression-here)' | npm start
```

Comment:
Why don't we change the `(expression-here)` into something useful?

How about some `JSONParse(HTTP(GET("knowsheet.io/uptime")).body).uptime.uptime_total_d`? I know it's long and not yet working, but the latter can/will be fixed, and ... you got the idea :)